Backpack/src/main/java/net/skyscanner/backpack/calendar/view/MonthView.kt

```diff
@@ -432,6 +439,14 @@ internal class MonthView @JvmOverloads constructor(
       y.toFloat(),
       monthNumberPaint
     )
+
+    labelsViewModel.draw(
```

Comment:
Will draw nothing if the labels not set, or will draw a dash if the labels are set, but there's no data for the current date.